app/src/main/java/com/devshish/internship/presentation/ui/utils/SnackbarExt.kt

```diff
@@ -0,0 +1,11 @@
+package com.devshish.internship.presentation.ui.utils
+
+import android.view.View
+import com.devshish.internship.R
+import com.google.android.material.snackbar.Snackbar
+
+fun Snackbar.set(action: View.OnClickListener?) {
```

Comment:
lambda instead of View.OnclickListener
+ move R.id.bottomNavView to params (you can make default param so not to put it everywhere)